gradle/libs.versions.toml

```diff
@@ -13,6 +13,7 @@ ktlint_gradle = "11.5.0"
 min_sdk_version = "14"
 mockito_core = "5.4.0"
 recyclerview = "1.3.0"
+viewpager2 = "1.1.0-beta02"
```

Comment:
Should we hold till 1.1.0 stable?

Reply:
Idk really, ViewPager2 `1.1.0-beta2` was released at the end of May 2023, and `1.1.0-beta` on August 4, 2021. We may have to wait a lot more...
And on the bright side, Google's beta releases are stable and well-tested. 

We could also base this on ViewPager `1.0.0` stable (released in 2019) since there are not breaking changes, but looking at the changelog I can spot a lot of useful bugfixes since then. What do you think @cortinico?

Reply:
I'm in for waiting 1.1.0 stable frankly. We could also merge it as it is, and don't release till stable is out